index.php

```diff
@@ -0,0 +1,88 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ *  A page for testing realtime event pushing form CL.
+ *
+ * @package    tool_realtime
+ * @copyright  2020 Daniel Conquit, Matthew Gray, Nicholas Parker, Dan Thistlewaite
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+require_once(dirname(__FILE__) . '/../../../config.php');
+
+$url = new moodle_url('/admin/tool/realtime/plugin/phppoll/');
+$PAGE->set_url($url);
```

Comment:
There is a 'sugar' syntax way of adding a report, see an example here to add the report to the admin navigation tree:

https://github.com/catalyst/moodle-tool_objectfs/blob/master/settings.php#L46-L48

And here in the report page to link it together:

https://github.com/catalyst/moodle-tool_objectfs/blob/master/object_status.php#L30

This also enforces the security so that only site admins can see this report